ai_genomics/analysis/ai_genomics_experts/patent_assignees.py

```diff
@@ -0,0 +1,62 @@
+from ai_genomics.getters.patents import get_ai_genomics_patents
+from ai_genomics import PROJECT_DIR
+import pandas as pd
+
+EXPERTS_PATH = PROJECT_DIR / "outputs/data/experts/"
+
+
+def assignee_string_to_list(patents: pd.DataFrame) -> pd.DataFrame:
+    """Turn string values in column into list
+    For example, string "['Massachusetts Institute Of Technology' 'Peck, David, D.']"
+    becomes list ['Massachusetts Institute Of Technology', 'Peck, David, D.']
+    """
+    patents["assignee"] = (
+        patents["assignee"]
+        .str.replace("'s", "s", regex=False)
+        .str.split("'")
+        .apply(
+            lambda assignee: [
+                token for token in assignee if token not in ["[", " ", "\n ", "]"]
+            ]
+        )
+    )
```

Comment:
I think you can just use ast.literal_eval to convert these string values into a list:

```suggestion
    patents["assignee"] =  patents['assignee'].apply(lambda x: ast.literal_eval(x))
```

Reply:
although this won't work if its an NA, so I'd drop any nans in the assignee col

Reply:
`ast.literal_eval` doesn't work underfortunately.

For example if we have this string: `"['The Procter & Gamble Company' 'Rui Li' 'Shi Huang' 'Tao He' 'Jiquan Liu'\n 'Jian Xu']"`

with `ast.literal_eval` it becomes `['The Procter & Gamble CompanyRui LiShi HuangTao HeJiquan LiuJian Xu']`

and with my function it becomes `['The Procter & Gamble Company',
 'Rui Li',
 'Shi Huang',
 'Tao He',
 'Jiquan Liu',
 'Jian Xu']`

Reply:
omg this is why!!!!